### ISSUE TITLE
GROUP BY and ORDER BY has the same object

### ISSUE THREAD

**Issue by dwips** [JavaScriptor/js-sql-parser#31]
Hi this is a great lib.

I found out that the GROUP BY clause is parsed value identical to ORDER BY clause which has sortOpt ASC or DESC

i think this is a bug because the group by clause also accepting sortOpt

![image](https://user-images.githubusercontent.com/6791049/75548868-ccf21980-5a60-11ea-8c40-6098b0ceef49.png)

![image](https://user-images.githubusercontent.com/6791049/75548931-fc088b00-5a60-11ea-90b9-b9081af8e09a.png)

Thank you

**Comment by albin3** [comment]
@dwips hi dwips, thanks for the nice and detailed bug report.

But it's not a bug. The grammar in https://dev.mysql.com/doc/refman/5.7/en/select.html show that `group by` grammar has sort options in Mysql version 5.7.

I also found that this grammar has been removed since Mysql version 8.0, follows https://dev.mysql.com/worklog/task/?id=8693 .

I think I'll keep this grammar in this repo since entire other grammar are follows 5.7 Mysql grammar.

This issue is closing and thanks again for reporting.
